datafusion/common/src/scalar/mod.rs

```diff
@@ -652,8 +655,8 @@ fn partial_cmp_map(m1: &Arc<MapArray>, m2: &Arc<MapArray>) -> Option<Ordering> {
         let arr1 = m1.entries().column(col_index);
         let arr2 = m2.entries().column(col_index);
 
-        let lt_res = arrow::compute::kernels::cmp::lt(arr1, arr2).ok()?;
-        let eq_res = arrow::compute::kernels::cmp::eq(arr1, arr2).ok()?;
+        let lt_res = lt(arr1, arr2).ok()?;
+        let eq_res = eq(arr1, arr2).ok()?;
```

Comment:
I actually wouldn't mind (and perhaps would even prefer) to keep this particular piece of code as it was.
The bare `lt` isn't as self-explanatory.

Can we somehow exempt arrow compute kernels from the check?